Gemfile

```diff
@@ -8,6 +8,8 @@ gem 'haml', '~> 4.0.6'
 gem 'grape', '~> 0.11.0'
 gem 'grape-entity', '~> 0.4.5'
 gem 'tutum', '~> 0.2.6'
+gem 'sqlite3'
+gem 'sequel'
```

Comment:
You should add the versioning info to these. Don't forget to re-run `bundle update` or `bundle install` after that.